src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Formatting/Rules/IndentBlockFormattingRule.cs

```diff
@@ -38,9 +41,8 @@ public override void AddIndentBlockOperations(List<IndentBlockOperation> list, S
 
         private void AddTypeParameterConstraintClauseOperation(List<IndentBlockOperation> list, SyntaxNode node)
         {
-            if (node is TypeParameterConstraintClauseSyntax typeParameterConstraintClause)
+            if (node is TypeParameterConstraintClauseSyntax { Parent: { } declaringNode })
```

Comment:
am not a fan.   :-/   .  would personally prefer `node is TypeParameterConstraintClauseSyntax clause && clause.Parent != null` tbh.  But not blocking on this.

Reply:
or just do the `RoslynDebug.AssertNotNull(clause .Parent);` like elsewhere.

Reply:
I'm hypothesizing this is a lack of familiarity more than anything, and once we use it for a bit the behavior will start to stand out as soon as we see it.